tools/chapel-py/setup.py

```diff
@@ -68,6 +103,12 @@
     "-lChplFrontendShared",
 ]
 
+if chpl_install_lib_path is not None:
+    LDFLAGS += [
+        "-L{}".format(chpl_install_lib_path),
+        "-Wl,-rpath,{}".format(chpl_install_lib_path),
+    ]
+
```

Comment:
Does this not mean that we are now embedding 2 rpaths? one for the source directory and one for the installed directory?

Reply:
yes, and I believe even more are added

Reply:
Should it be that way? We are embedding rpaths that we never intend to use, which seems like an issue to me

Reply:
The python build system seems to need the rpath during the build, and we need a different one later after install, so unless we go back to using `chrpath` or other post build binary manipulation, I think we're stuck with it :(

Reply:
Ugh yeah its required to generate the pyi files. Ok well thats very dissatisfying and is going to be a pain elsewhere (fedora namely, although the chrpath stuff should continue to work around this). But its fine for this PR

Reply:
The Linux loader should ignore RPATH directories that are inaccessible (or don't contain a matching *.so), so extraneous rpath entries should be "mostly harmless".

On the other hand, relying on `chrpath` at install time is a fragile design in general, because usually `chrpath` [can only replace an RPATH with one of equal or shorter length](https://man.cx/chrpath#:~:text=can%20only%20replace%20an%20RPATH%20with%20one%20of%20equal%20or%20shorter%20length). So installing linked executables to a sufficiently long install prefix might "just break" if that insertion is delayed until a post-link `chrpath` step.

Reply:
The Linux loader should ignore extraneous RPATH entries, but some of the Linux (namely Fedora) packaging stuff does complain if you have unused/dead/broken RPATH entries.

Reply:
Well `chrpath` should always be safe to _remove_ unused RPATH entries at install time, it's just fragile to assume that it can always successfully replace/insert arbitrary entries.

Reply:
Yes, that is what I said [here](https://github.com/chapel-lang/chapel/pull/26388#discussion_r1879175610). I am ok with these dead paths, since chrpath is already being used to rewrite RPATHs in the package builds.

Reply:
> some of the Linux (namely Fedora) packaging stuff does complain if you have unused/dead/broken RPATH entries

I ran into this in the spack builds (on some linux) until I fixed the formatting of the previously existing LDFLAGS (per the diff you slacked me at one point). Just curious if that might be the same symptom or something else.

Reply:
Possibly related, although I have had issues with other binaries besides just chapel-py